### PR TITLE
Clone files in inlines

### LINKED ISSUE #11: Blank FileField in inlines
I've got "This field is required." on FileField inside inlines while saving clone.
Actually, also I get blank FileField in the main formset of my model.
I use Grappelli, but I've reproduced the same problem in standard admin interface.

### PR DESCRIPTION
**Pull request by ghost**
Fixes #11 

Unlike @desecho's PR at #27, it doesn't depend on the inline relation to have exactly the same name as the file field with a "s" appended to it (in other words, #27 doesn't use `prefix` when it should).

Also, include tests.

### CODE REVIEW

File: modelclone/admin.py [hunk 1]
```diff
@@ -107,7 +107,18 @@ def clone_view(self, request, object_id, form_url='', extra_context=None):
...
+
+                request_files = request.FILES
+                filter_params = {'%s__pk' % original_obj.__class__.__name__.lower(): original_obj.pk}
+                inlined_objs = inline.model.objects.filter(**filter_params)
+                for n, inlined_obj in enumerate(inlined_objs.all()):
+                    for field in inlined_obj._meta.fields:
+                        if isinstance(field, FileField) and field not in request_files:
+                            value = field.value_from_object(inlined_obj)
+                            file_field_name = '{}-{}-{}'.format(prefix, n, field.name)
+                            request_files[file_field_name] = value
```
Comment (igorsobreira) [inline modelclone/admin.py]: Thanks @hsoft!
I just found one issue using it with the `sampleproject` locally, here is how to reproduce:

1. Create a `Post` with an image, let's say `image1.jpg`
2. Click Clone
3. Before saving the duplicate, select a new image, like `image2.jpg`, and Save
4. The new duplicated post will still have `image1.jpg`

I think if you use `setdefault()` in `request_files` here should solve it

**Review by igorsobreira — CHANGES_REQUESTED** [review]
Looks good @hsoft, I this was the only issue I found

File: modelclone/admin.py (hunk 1 above)
Comment (hsoft) [reply modelclone/admin.py]: I'll add a test case for this, probably tomorrow.

**Comment by hsoft** [comment]
@igorsobreira fixed

**Comment by igorsobreira** [comment]
Thanks @hsoft! New version 0.7.1 is on pypi

**Pull request merged**